src/main/resources/db/migration/V0.0.1__create_set_table.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE
+    IF NOT EXISTS "tbl_set" (
+        "set_id" VARCHAR(36) UNIQUE NOT NULL,
+        "ygojson_id" VARCHAR(36) UNIQUE,
+        "print_number_prefix" VARCHAR(255),
```

Comment:
Open ticket to refine in the future (with lower priority) the VARCHAR length of the entity definitions (as this can have an impact in performance)

Reply:
Open #157